README.md

```diff
@@ -21,893 +17,3 @@ Goose is a Rust load testing tool inspired by [Locust](https://locust.io/). User
   - [Real-life load testing with Goose](https://www.tag1consulting.com/blog/real-life-goose-load-testing)
   - [Gaggle: a distributed load test](https://www.tag1consulting.com/blog/show-me-how-flock-flies-working-gaggle-goose)
   - [Optimizing Goose performance](https://www.tag1consulting.com/blog/golden-goose-egg-compile-time-adventure)
```

Comment:
Add some simple documentation here about how to build the book, for example:

> Manually build and view "The Goose Book" with [mdBook](https://rust-lang.github.io/mdBook/) with the following command:
> ```
>     mdbook build src/docs/goose-book --open
> ```

Reply:
We opened a RFC and there was discussion around a skeleton when this wasn't mentioned as 'nice to have' additions. Resolving as can't fix in 3-6 months.